OptionSelector/CategoryMain.h

```diff
@@ -0,0 +1,16 @@
+//
+//  CategoryMain.h
+//  OptionSelector
+//
+//  Created by Charles Kang on 8/11/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "Categories.h"
+
+@interface CategoryMain : Categories
```

Comment:
I don't think this needs to extend "Categories" (I think it's maybe confusing because of the name).